index.js

```diff
@@ -14,14 +26,37 @@ function sortByStringLength() {}
  * Example: "Hello"
  * [ 'elloH', 'lloHe', 'loHel', 'oHell', 'Hello' ]
  */
-function textScroller() {}
+function textScroller(word) {
+  if (word === '') {
+    return [];
+  }
+
+  const scrollingPositions = [];
+
+  for (let i = 1; i <= word.length; i++) {
+    const scrolledWord = word.slice(i) + word.slice(0, i);
```

Comment:
Nice work - keep this pattern in mind.  It's used with something called a 'stack' where we 'pop' the top item off of the stack or list and then do some sort of operation on it.  This process continues until we've moved through the entire stack.